sign_node/signer.py

```diff
@@ -29,15 +34,27 @@
 import pgpy
 
 from sign_node.errors import SignError
-from sign_node.utils.file_utils import download_file, hash_file, safe_mkdir
+from sign_node.utils.file_utils import (
+    download_file,
+    hash_file,
+    safe_mkdir,
+)
 from sign_node.utils.codenotary import Codenotary
 from sign_node.uploaders.pulp import PulpRpmUploader
-from sign_node.package_sign import (
-    sign_dsc_package, sign_deb_package, sign_rpm_package
-)
+from sign_node.package_sign import sign_rpm_package
 
 
-__all__ = ["Signer"]
+__all__ = ['Signer']
+
+gpg_scenario_template = (
+    '%no-protection\n'
+    'Key-Type: RSA\n'
+    'Key-Length: 2048\n'
+    'Subkey-Type: default\n'
+    'Subkey-Length: 2048\n'
+    'Name-Real: {sign_key_uid}\n'
+    'Expire-Date: 0\n'
+)
```

Comment:
2048 bits might be a bit short, especially if the keys never expire

Reply:
can we move it to the constants?

Reply:
Resolved